autoendpoint/src/extractors/subscription.rs

```diff
@@ -71,7 +91,10 @@ impl FromRequest for Subscription {
 
             // Validate the VAPID JWT token and record the version
             if let Some(vapid) = &vapid {
-                validate_vapid_jwt(vapid)?;
+                validate_vapid_jwt(
+                    vapid,
+                    &format!("{}://{}", &state.settings.scheme, &state.settings.host),
```

Comment:
Ah cool, we already have that method.

It does not hide the standard ports though, can we fix that here? Or making it a separate issue might make more sense in case it affects something else.

autopush [already has logic for this](https://github.com/mozilla-services/autopush-rs/blob/5badbfb/autopush/src/settings.rs#L116).

Reply:
Oh, cool. Didn't know about that function.

I'll make a separate issue to fix that. https://github.com/mozilla-services/autopush-rs/issues/229